Model/Checkout/PlaceOrder.php

```diff
@@ -46,6 +47,7 @@ class PlaceOrder extends AmwalCheckoutAction
     private GetAmwalOrderData $getAmwalOrderData;
     private SentryExceptionReport $sentryExceptionReport;
     private SearchCriteriaBuilder $searchCriteriaBuilder;
+    protected StoreManagerInterface $storeManager;
```

Comment:
This can (should) be private since we're limiting it's usage to the current class
```suggestion
    private StoreManagerInterface $storeManager;
```